SPOOP/Assets/Scripts/Level1/ObstacleController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObstacleController : MonoBehaviour 
+{
+    public float speed;
+
+    void Start () 
+    {
+        GameObject.FindWithTag ("Player").GetComponent<PlayerController>().obstacle = gameObject;
```

Comment:
I like this but i think we will still run into issues with incorporating multiple obstacles in a level (which I foresee us doing).